terraform/ecs/main.tf

```
@@ -141,6 +141,21 @@ resource "aws_autoscaling_group" "clusterasg" {
   }
 }
 
+data "aws_instances" "ecs-instances" {
+  instance_tags = {
+    Patch = var.patch
+  }
+
+  filter {
+    name   = "tag:Name"
+    values = ["Integ-test-ecs-instance"]
+  }
+
+  instance_state_names = ["running", "stopped"]
```

```suggestion
  instance_state_names = ["running", "stopped"]
```
Why do we filter on both `running` and `stopped` rather than just `running`?

---

I carried it over from the example.  I think just `running` makes more sense if this case.